package.json

```diff
@@ -15,41 +15,22 @@
     "build": "tsdx build",
     "test": "tsdx test --passWithNoTests",
     "lint": "tsdx lint",
-    "prepare": "tsdx build",
     "size": "size-limit",
     "analyze": "size-limit --why",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "prepare": "husky install"
   },
```

Comment:
I found prettier configuration file but we do not have a command to run `prettier`. 
and how prettier will run according to the config file.

Reply:
We have a plugin for prettier integrated in eslint which will format the staged files on precommit hook

Reply:
actually I tried it but I got an error 😂.
```
@elhaw ➜ /workspaces/m3ntorship-ui (initial-project-setup) $ git commit -m "add line"
.husky/pre-commit: 4: lint-staged: not found
husky - pre-commit hook exited with code 127 (error)
husky - command not found in PATH=/usr/local/libexec/git-core:/home/codespace/.rbenv/shims:/vscode/bin/linux-x64/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/bin/remote-cli:/home/codespace/.nvm/versions/node/v14.19.3/bin:/home/codespace/.npm-global/bin:/home/codespace/.python/current/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/codespace/.dotnet:/home/codespace/.dotnet/tools:/usr/local/sdkman/bin:/usr/local/sdkman/candidates/gradle/current/bin:/usr/local/sdkman/candidates/java/current/bin:/opt/maven/lts:/usr/local/go/bin:/go/bin:/usr/local/py-utils/bin:/opt/conda/condabin:/home/codespace/.java/current/bin:/home/codespace/.nodejs/current/bin:/home/codespace/.php/current/bin:/home/codespace/.ruby/current/bin:/home/codespace/.maven/current/bin:/home/codespace/.hugo/current/bin:/home/codespace/.local/bin:/opt/oryx:/opt/nodejs/lts/bin:/opt/dotnet/lts:/opt/python/latest/bin:/opt/php/lts/bin:/opt/php-composer:/opt/yarn/stable/bin:/opt/hugo/lts::/opt/java/lts/bin:/opt/maven/lts/bin:/opt/ruby/lts/bin:/usr/local/rvm/bin
```